src/components/DocsLink/DocsLink.tsx

```diff
@@ -1,46 +1,24 @@
-import React, { type ReactNode } from 'react';
-import { GrafanaTheme2 } from '@grafana/data';
-import { Icon, useStyles2 } from '@grafana/ui';
-import { css, cx } from '@emotion/css';
+import React, { type ReactNode, useCallback } from 'react';
+import { TextLink } from '@grafana/ui';
 
-const docs = {
-  probes: `https://grafana.com/docs/grafana-cloud/testing/synthetic-monitoring/create-checks/public-probes/`,
-  publicProbes: `https://grafana.com/docs/grafana-cloud/testing/synthetic-monitoring/create-checks/public-probes/`,
-  privateProbes: `https://grafana.com/docs/grafana-cloud/testing/synthetic-monitoring/set-up/set-up-private-probes/`,
-  addPrivateProbe: `https://grafana.com/docs/grafana-cloud/testing/synthetic-monitoring/set-up/set-up-private-probes/#add-a-new-probe-in-your-grafana-instance`,
-};
+import { onDocsLinkClick } from 'components/DocsLink/DocsLink.utils';
 
 type DocsLinkProps = {
-  article: keyof typeof docs;
   children: ReactNode;
-  iconPosition?: 'prefix' | 'suffix';
-  className?: string;
+  href: string;
+  inline?: boolean;
+  source: string;
```

Comment:
I think that the "source" prop warrants a description to what it's expected to be